Sources/MapboxMaps/Style/Types/Expression.swift

```diff
@@ -17,20 +17,34 @@ public struct Expression: Codable, CustomStringConvertible, Equatable {
 
     /// The operator of this expression
     public var `operator`: Operator {
-        guard let first = elements.first, case Element.operator(let op) = first else {
-            fatalError("First element of the expression is not an operator.")
+        var firstElement = elements.first
+        /// If the expression starts with an argument instead of an operator, return the first operator of a contained expression if available.
+        if case let .argument(firstArgument) = firstElement,
+           case let .expression(firstExpression) = firstArgument {
+            firstElement = firstExpression.elements.first
+        }
+
+        guard let first = firstElement, case Element.operator(let op) = first else {
+            fatalError("First element of the expression is not an operator nor another expression.")
         }
         return op
```

Comment:
nit: can we shorten like this?
```suggestion
        switch elements.first {
            case .operator(let op): return op
            case .argument(.expression(let expression)): return expression.operator
            default:
                fatalError("First element of the expression is not an operator nor another expression.")
        }
```

Reply:
though I'm still a little bit concern with this approach, it doesn't make much sense to me that the Operator of an operatorless Expression is the first argument's operator. Could we update the doc of this computed property to reflect this change at least?

Reply:
Agreed that it is a bit strange. An alternative approach to avoid the fatalError could be to return an empty `Operator`. 

> Could we update the doc of this computed property to reflect this change at least?
Yes, can do